mifospay/src/main/res/layout/activity_link_bank_account.xml

```diff
@@ -1,123 +1,143 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
we don't need this xml delete it and directly use composable, please see other example.